src/lib/fragments/Igv.react.js

```diff
@@ -0,0 +1,61 @@
+import React, {Component} from 'react';
+import {propTypes, defaultProps} from '../components/Igv.react';
+import igv from 'igv';
+
+/**
+ * The Igv component is an interactive genome visualization component
+ * developed by the Integrative Genomics Viewer (IGV) team. It uses an
+ * example integration of igv.js and React (https://www.npmjs.com/package/igv).
+ */
+export default class Igv extends Component {
+    componentDidMount() {
+        var igvContainer = document.getElementById(this.props.id);
```

Comment:
Don't use `getElementById` - use a `ref` instead. For an example, see https://github.com/plotly/dash-core-components/pull/604/files where this change was made to `dcc.Graph`

Reply:
Changed in 0e650d0.